Source/WebKit/Platform/spi/Cocoa/SafeBrowsingSPI.h

```diff
@@ -70,9 +71,12 @@ WTF_EXTERN_C_END
 + (SSBLookupContext *)sharedLookupContext;
 
 - (void)lookUpURL:(NSURL *)URL completionHandler:(void (^)(SSBLookupResult *, NSError *))completionHandler;
+- (void)lookUpURL:(NSURL *)URL isMainFrame:(bool)isMainFrame hasHighConfidenceOfSafety:(BOOL)hasHighConfidenceOfSafety completionHandler:(void (^)(SSBLookupResult *, NSError * _Nullable))completionHandler;
```

Comment:
I think we need a test that swizzles this and responds immediately with a phishing result and verify it shows a warning and with a nothing result and verify it loads without showing a warning.

Reply:
I think those two cases are already covered by `SafeBrowsing WKWebViewGoBack`